Schutzfile

```diff
@@ -2,7 +2,7 @@
   "global": {
     "dependencies": {
       "osbuild": {
-        "commit": "b87eaf603284681335704910636123778dd57895"
```

Comment:
Do the new distributions needs an updated version of OSBuild ?

Reply:
Yes I believe so because otherwise there would be no osbuild rpms avaiable for 8.8 and 9.2

Reply:
Normally this field would be updated automatically by OSBuild's bot. In this particular case though it makes sense to update it by hand. Thanks.